### PR TITLE
Micu/#7 map overview

### PR DESCRIPTION
**Pull request by micuintus**
One thing related to this PR, which I guess has to follow soon: A 'recenter' icon in the mao overview,
which resets the the zoom and puts the current location back to the center.

### CODE REVIEW

**Comment by jmastr** [comment]
First of all, this looks super nice! Especially the different colors for vegan, vegetarian, omnivore. Great job.

Without looking too much into the source code, I noticed two things:
1. If one selects a venue via the map overview, he/she can then not get directions from the venue details page.
1. If one selects a venue via the list view and gets directions, the marker is always red and not green/orange/red like in the map overview (which is really, really nice!)

File: qml/components-v-play/Theme.qml [hunk 1]
```diff
@@ -46,9 +52,7 @@ QtObject {
         case "about":
             icon = "info";
             break;
-        case "locationarrow":
```
Comment (jmastr) [inline qml/components-v-play/Theme.qml]: This diff is unnecessary.  Leave `locationarrow` there and just put `map` underneath it.

**Comment by jmastr** [comment]
One thing i also noticed is, that it takes some time (on an iPhone 6s) until the map markers appear, which makes the user guessing, what's going on.

**Comment by micuintus** [comment]
> First of all, this looks super nice! Especially the different colors for vegan, vegetarian, omnivore. Great job.

Thanks. 💯 

>    If one selects a venue via the map overview, he/she can then not get directions from the venue details page.

Well, that was on purpose B/C I thought it would be weird to have 2 map pages on the navigation stack:
list page -> map overview page -> venue description -> venue map page

>     If one selects a venue via the list view and gets directions, the marker is always red and not green/orange/red like in the map overview.

OK. ATM the map overview page and the venue map page are actually two separate pages, but we could add the vegan category color to the venue map page if you want.

In the long/mid run we should merge the venue map page and the map overview page, but there some unsolved UX questions: Gray out other items? How does the navigation look like? Etc.

**Comment by jmastr** [comment]
Ok. We can address everything in #149.

**Pull request merged**